layouts/page/swagger-ui.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <title>Editoast REST API</title>
```

Comment:
Nit: Gateway openAPI will also be called Editoast

Reply:
Good catch, fixed!